pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -320,129 +320,129 @@
-<!ENTITY pbSourceViewThemes.label "Theme">
-<!ENTITY compositionGoals.tooltip "Show or hide the Composition Goals status bar">
-<!ENTITY switchBetweenNormalAndCode.tooltip "Switch between Normal and Code views">
-<!ENTITY compositionGoals.label "Goals">
+<!ENTITY pbStatsChars.label "caracteres:">
+<!ENTITY formattingToolbar.label "Barra de ferramentas">
+<!ENTITY pbShowCompositionGoalsToolbarCmd.label "Barra de status do objetivo">
+<!ENTITY pbStatsTime.label "horário:">
+<!ENTITY pbSourceViewThemes.label "Tema">
+<!ENTITY compositionGoals.tooltip "Mostrar ou ocultar a barra de status de Objetivos de Composição">
+<!ENTITY switchBetweenNormalAndCode.tooltip "Alternar entre modos Normal e Código">
+<!ENTITY compositionGoals.label "Objetivos">
 <!ENTITY tagButton.key "T">
-<!ENTITY pbSourceView.label "Code View">
+<!ENTITY pbSourceView.label "Exibição de Código">
 <!ENTITY pbShowCompositionGoalsToolbarCmd.accesskey "G">
-<!ENTITY formattingToolbar.tooltip "Show or hide the toolbar">
+<!ENTITY formattingToolbar.tooltip "Mostrar ou ocultar a barra de ferramentas">
 
-<!ENTITY pbCreateResponse.label "Create Response...">
-<!ENTITY pbSearchGoogle.label "Search in Google">
+<!ENTITY pbCreateResponse.label "Criar resposta...">
+<!ENTITY pbSearchGoogle.label "Pesquisar no Google">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY templatePlaceholderLastName.label "Last Name">
-<!ENTITY templatePlaceholderToRecipients.label "To Recipients">
-<!ENTITY templatePlaceholderCcRecipients.label "Cc Recipients">
-<!ENTITY templatePlaceholders.tooltiptext "Insert a Template Placeholder">
-<!ENTITY messageSize.label "Message size:">
-<!ENTITY templatePlaceholderFirstName.label "First Name">
-<!ENTITY templatePlaceholderEmail.label "Email Address">
-<!ENTITY templatePlaceholderMessageId.label "Message ID">
-<!ENTITY templatePlaceholderCustom.label "Custom Placeholder">
-<!ENTITY templatePlaceholderThisMessage.label "This Message">
-<!ENTITY imageSize.label "Image size:">
-<!ENTITY largeImageResize.label "Large">
-<!ENTITY templatePlaceholderMySelectedIdentity.label "My Selected Identity">
-<!ENTITY templacePlaceholderFillValues.label "Fill Values">
-<!ENTITY templatePlaceholderSourceMessage.label "Source Message">
-<!ENTITY templatePlaceholderSender.label "Sender">
-<!ENTITY mediumImageResize.label "Medium">
-<!ENTITY templatePlaceholderSubject.label "Subject">
-<!ENTITY smallImageResize.label "Small">
-<!ENTITY templatePlaceholderIdentity.label "Full Name">
-<!ENTITY templatePlaceholderDate.label "Date">
+<!ENTITY templatePlaceholderLastName.label "Sobrenome">
+<!ENTITY templatePlaceholderToRecipients.label "Destinatários Para>
+<!ENTITY templatePlaceholderCcRecipients.label "Destinatários Cc">
+<!ENTITY templatePlaceholders.tooltiptext "Inserir um modelo placeholder">
+<!ENTITY messageSize.label "Tamanho da mensagem:">
+<!ENTITY templatePlaceholderFirstName.label "Nome">
+<!ENTITY templatePlaceholderEmail.label "Endereço de e-mail">
```

Comment:
@brunotiussi Understood. After receiving Sherman's approval, either use e-mail or email (chose one to keep consistency).